fundamental-concepts/data-types-imports/6_stdlib.md

```diff
@@ -0,0 +1,73 @@
+# Importing modules 2/2: Importing the standard library
+
+The same way you import your own module, you can import modules from the [standard
+library](https://docs.daml.com/daml/stdlib/index.html). Let's import the `DA.Time` module and add a
+birthday date to our `Person` data type in `daml/AddressBook.daml`{{open}}:
+
+<pre class="file" data-target="clipboard">
+import DA.Date
+</pre>
+
+Now we can extend the `Person` record in `daml/AddressBook`{{open}}
+
+<pre class="file" data-target="clipboard">
+data Person = Person {
+  name: Text,
+  address: Address,
+  age: Int,
+  birthday: Date
+}
+</pre>
+
+And add the missing field to `testPerson`:
+
+<pre class=file data-target="clipboard">
+testPerson: Person
+testPerson = Person {name = "Alice", address = Address {street = "RabbitStreet", city = "TheBigUnknown", country = "Wonderland"}, age = 7, birthday = date 30 Oct 2013}
+</pre>
+
+The functions `date` and the constructor `Oct` are provided by the `DA.Date` module of the standard
+library. In the next steps we look at some of the most commonly used data types and functions
+provided by the standard library.
+
+The same way you import the standard library you can import modules of your own packages. You can do
+this in three easy steps:
+
+1. First compile the package with the module you want to import. In our case this would be the
+   `organizer` package 
+
+```
+cd organizer
+daml build
+```{{execute T1}}
```

Comment:
A minor thing but the numbering doesn't work (it doesn't continue). A suggestion from [SO](https://stackoverflow.com/questions/18088955/markdown-continue-numbered-list) how to fix it. Not sure if it'll work for < pre > tag.

Reply:
fixed.